addons/field_rations/stringtable.xml

```diff
@@ -205,6 +205,12 @@
             <Russian>динамический</Russian>
             <Spanish>Dinámica</Spanish>
         </Key>
+        <Key ID="STR_ACEX_Field_Rations_WaterSupply_3DEN_DisplayName">
+            <English>Water Supply</English>
+        </Key>
+        <Key ID="STR_ACEX_Field_Rations_WaterSupply_3DEN_Description">
+            <English>The amount of water available for ACEX Field Rations water source actions (-1 disabled, -10 infinite)</English>
```

Comment:
Why not 0 disabled, -1 infinite. :thinking: 

That was a weird decision, but not really related to this PR.

Reply:
Yeah it is weird. I guess it was to have the `Check Remaining Water` action show `No water remaining` when it is zero rather than no actions at all.